src/question/mod.rs

```diff
@@ -431,6 +434,39 @@ impl Question<'static> {
         MultiSelectBuilder::new(name.into())
     }
 
+    /// Prompt that allows the user to organize a list of options.
+    ///
+    /// The choices are represented with the [`Choice`] enum. [`Choice::Choice`] can be multi-line.
+    /// but [`Choice::Separator`]s will be ignored.
```

Comment:
Instead of deleting these two lines, i suggest replace these with a comment like "Due to the kind of question, the separators will be ignored"

Is it ok ?

Reply:
I have a feeling I didn't convey my point across properly. If you see [this comment](https://github.com/Lutetium-Vanadium/requestty/pull/16#discussion_r950843068), the use of `Choice` should not be visible to the user. Therefore, using the `Choice` enum in the documentation may be confusing. Do you get what I mean?

To keep the part about multi-line, you can say that "The choices are [\`String\`]s and can be multiline"